### DISCUSSION TITLE
Adding text for the TextureButton

### DISCUSSION THREAD

**Discussion by piotrtrkostrzewski94** [godotengine/godot-proposals#9405] [2D]
Good morning.
Today, when creating a button using TextureButton, I noticed that, unlike Button, it is not possible to add text to the button directly in Godot.  I know that you can add it in a graphics program, but is it possible to add the option of entering text directly in Godot (just like for Button)?  I think this could be solved by combining the code for the TextureButton in the Inspector with the appropriate code fragment responsible for adding text for the Button in the Inspector.  Thank you very much for your time.  Have a nice day and Happy Easter.

**Comment by theraot** [comment]
You should be able to work around this by adding a `Label` child of the `TextureButton`.

**Reply by piotrtrkostrzewski94** [reply]
Thank you very much.
I will try this solution.
Have a Happy Easter
